src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -289,4 +350,78 @@ protected function args_to_context( array $arguments, Context $context ) {
 
 		return $context_args;
 	}
+
+	/**
+	 * Handles gathering the data for admin fields.
+	 *
+	 * @since TBD
+	 *
+	 * @param int                 $field_name    The ID of the field.
+	 * @param array<string,mixed> $field       The field info.
```

Comment:
:no_entry_sign: Doc comment for parameter $field does not match actual variable name $field_name

<sub>hash: 08898b45eb1755e36c56f21ecaa53201</sub>